server/scripts/expire-participants.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-console, no-restricted-syntax, no-await-in-loop */
+const { dbClient } = require('../db');
+const {
+  createOpenParticipantsTable,
+  getExpiredParticipants,
+  dropOpenParticipantsTable,
+  expiredParticipantsStatusInvalidation,
+} = require('../services/clean-up');
+/**
+ * Script entry method (main)
+ */
+(async () => {
+  try {
+    console.log('Starting expiring script...');
+    await dbClient.connect();
+    console.log(`Connected to database`);
+    await createOpenParticipantsTable();
+    const report = await getExpiredParticipants();
+    if (report.length > 0) {
+      console.log(`Withdrawing ${report.length} open, expired participants`);
+      console.table(report);
+    } else {
+      console.log('No expired open participants found');
+    }
+    await expiredParticipantsStatusInvalidation();
```

Comment:
Do we still want to run this if `report.length === 0`?

Reply:
I don't love having two cleanup jobs named with synonyms, how is `expired` different from `stale`?

Can we come up with naming that differentiates the two?

maybe `createStaleOpenParticipantsTable`/`cleanStaleOpenParticipants` or something?

Reply:
Yeah, expired vs. stale, hard to distinct between the two even though there is a distinction.

I like cleanStaleOpen vs. cleanStaleInProgress (the existing job)